WORKSPACE

```diff
@@ -12,17 +12,16 @@ http_archive(
     sha256 = "d03625db67e9fb0905bbd206fa97e32ae9da894fe234a493e7517fd25faec914",
 )
 
+load("@io_bazel_rules_go//go:def.bzl", "go_repository")
+load("//:repositories.bzl", "repositories")
 load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_register_toolchains")
 
-go_rules_dependencies()
-
-go_register_toolchains()
 
-load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
-
-gazelle_dependencies()
-
-load("@io_bazel_rules_go//go:def.bzl", "go_repository")
+go_repository(
+    name = "org_golang_google_genproto",
+    commit = "383e8b2c3b9e36c4076b235b32537292176bae20",
+    importpath = "google.golang.org/genproto",
+)
```

Comment:
Sorry, could you reference `Gopkg.toml` here as well? These two must be kept in sync, obviously.

Reply:
Really these should all reference Gopkg.toml :thinking:

Reply:
Done!